SwitchItem/SwitchItem.module.less

```diff
@@ -4,11 +4,50 @@
 @import "../styles/skin.less";
 
 .switchItem {
+	.label {
+		vertical-align: middle;
+	}
+
```

Comment:
is it needed ? when I remove this It looks the same.

Reply:
It seems it looks fine without this style. I removed it